lib/honeybadger/honeybadger.rb

```diff
@@ -20,6 +20,7 @@ def notify_base(exception_or_opts, options = {})
 
     alias_method :notify_super, :notify
     alias_method :notify, :notify_base
+    alias_method :notify_or_ignore, :notify_base
```

Comment:
although honeybadger itself aliases notify_or_ignore to notify, it does it at a time where my notify is not yet defined, so I also have to do this.